lib/DB/Schema/Result/User.pm

```diff
@@ -124,7 +124,7 @@ __PACKAGE__->inflate_column(
 __PACKAGE__->set_primary_key('user_id');
 __PACKAGE__->add_unique_constraint([qw/username/]);
 
-__PACKAGE__->has_many(course_users => 'DB::Schema::Result::CourseUser', { 'foreign.user_id' => 'self.user_id' });
+__PACKAGE__->has_many(course_users => 'DB::Schema::Result::CourseUser', 'user_id');
```

Comment:
Why the change here?  These are equivalent, and what was wrong with the shorthand?

Reply:
isn't the changed version technically the 'shorthand'? When we have the same id on two tables, this is more concise?

Reply:
Hah.  It seems that I read the diff in reverse.  The changed version is the shorthand.  Never mind then.